tools/create-project.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Creates a stub LORIS project directory at the location passed as the first argument
+mkdir -p $1/data $1/libraries $1/instruments $1/templates $1/tables_sql $1/modules $1/smarty/templates_c
```

Comment:
FWIW, since you are using bash you could make it a bit shorter `mkdir -p $1/{data,libraries,instruments,...}`

Reply:
will there be more ?  i.e. permissions fixups and whatnot?

Reply:
yes, but minimally more (at least templates_c needs to be writable by the apache user)

I'm also not sure if the `composer install --no-dev` command (that needs to be executed in `/usr/share/loris`) goes here, or goes somewhere else in the packaging. The workflow's still a little muddy for me.

Reply:
you could call it after `dh_install` which would "install" all those listed in `debian/install` file... smth like
```make
override_dh_install:
    dh_install
    ( cd debian/tmp/usr/share/loris; composer; )
```
or may be before dh_install but right here since that is where composer lives

```
override_dh_install:
    composer
    dh_install
```
and add whatever directory composer "installs" under within that `debian/install`

Reply:
You're going to want to quote the arguments, otherwise spaces in the directory name will be unfortunate.

    for d in data libraries instruments templates tables_sql modules smarty/templates_c ; do
      mkdir -p "${1}/${d}"
    done

Reply:
Thanks for catching that @aqw! I updated it with the change.